test/unit/associations/belongs-to.test.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+/* jshint -W030 */
+var chai = require('chai')
+  , expect = chai.expect
+  , Support   = require(__dirname + '/../support')
+  , current   = Support.sequelize;
+
+describe(Support.getTestDialectTeaser('belongsTo'), function() {
+  it('should not override custom methods with association mixin', function(){
+    var methods = {
```

Comment:
Use `const` and one line per declaration. Tests have not been refactored yet, but new tests should use ES6